src/app/components/helperComponents/LinkList.jsx

```diff
@@ -182,7 +188,7 @@ export default compose(
           langtag={langtag}
           clickHandler={clickHandler}
           mouseOverHandler={{
-            box: () => null, //mouseOverBoxHandler,
+            box: setHoveredElement,
             item: () =>
               setHovered(isAttachment ? link.uuid : link.linkTarget.rowId)
```

Comment:
```
  const handleSelectedItem = useCallback(() => setHovered(isAttachment ? link.uuid : link.linkTarget.rowId), [isAttachment, link])
```

Reply:
Das funktioniert so nicht, da renderSortableLink keine react-component ist und ich in der LinkList Component keinen Zugriff auf link und isAttachment habe

Reply:
Naja, in den Statehandler schieben hat ja den selben Effekt :)